### PR TITLE
[DEVX-596]: Test Coverage Scoring and Tests Addition for App, Dataset, Input, Model Classes

### PR DESCRIPTION
**Pull request by srikanthbachala20**
### Why

* To calculate test coverage of SDK
* To increase test coverage
* Initially, the test coverage is 53%
* Now, the test coverage is increased by 2% to 55%

### How

* **pytest** library can be used to calculate the test coverage of SDK
* Addition of tests for App, Dataset, Input & Model classes to increase tests coverage

### CODE REVIEW

File: .github/workflows/run_tests.yml [hunk 1]
```diff
@@ -47,7 +47,7 @@ jobs:
         export CLARIFAI_USER_ID="$(python scripts/key_for_tests.py --get-userid)"
         export CLARIFAI_PAT="$(python scripts/key_for_tests.py --create-pat)"
         if [ "${{ github.event.pull_request.head.repo.full_name }}" = "${{ github.repository }}" ]; then
-          pytest tests/
+          pytest --cov=.
```
Comment (sanjaychelliah) [inline .github/workflows/run_tests.yml:50]: Just confirming, this command will run both the tests and coverage right?

File: tests/test_app.py [hunk 2]
```diff
@@ -181,6 +195,11 @@ def test_patch_model(self, create_app, caplog):
 
   def test_delete_dataset(self, create_app, caplog):
     with caplog.at_level(logging.INFO):
+      dataset = create_app.dataset(dataset_id=CREATE_DATASET_ID)
+      versions = list(dataset.list_versions())
+      print("Dataset Version", versions[0].version.id)
```
Comment (sanjaychelliah) [inline tests/test_app.py:200]: We can delete this I think!

**Review by sanjaychelliah — COMMENTED** [review]
Added a couple of comments.

File: .github/workflows/run_tests.yml (hunk 1 above)
Comment (srikanthbachala20) [reply .github/workflows/run_tests.yml:50]: Yes

File: tests/test_app.py (hunk 2 above)
Comment (srikanthbachala20) [reply tests/test_app.py:200]: Done

**Review by sanjaychelliah — APPROVED** [review]
👍

**Review by sanjaychelliah — APPROVED** [review]

**Review by nitinbhojwani — APPROVED** [review]

**Pull request merged**